spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -353,7 +353,7 @@ private DepInterpreter getDepInterpreter() {
   public boolean isYarnMode() {
     String master = getProperty("master");
     if (master == null) {
-      master = getProperty().getProperty("spark.master", "local[*]");
+      master = getProperty("spark.master", "local[*]");
     }
     return master.startsWith("yarn");
```

Comment:
will this be refactored to share code with SparkInterpreterLauncher.java?

Reply:
They are in different module (spark, zeppelin-zengine). so I am afraid there's no chance to reuse code.